api/src/main/java/org/pmiops/workbench/api/WorkspacesController.java

```diff
@@ -383,6 +384,29 @@ private FirecloudWorkspaceId generateFirecloudWorkspaceId(String namespace, Stri
       Workspace workspace) {
     org.pmiops.workbench.db.model.Workspace dbWorkspace = workspaceService.getRequired(
         workspaceNamespace, workspaceId);
+    String userAccess;
+    try {
```

Comment:
I think we're going to want to put this logic in a method somewhere and also call it in a bunch of other places (e.g. create / updating / reviewing cohorts, modifying notebooks).